### PR TITLE
Handle the ONNX opset for BatchNormalization

### PR DESCRIPTION
**Pull request by houseroad**
Remove consumed_inputs before converted to Caffe2 ops.

### CODE REVIEW

**Comment by houseroad** [comment]
@caffe2bot test this please

File: caffe2/onnx/backend.cc [hunk 1]
```diff
@@ -782,6 +783,17 @@ Caffe2Ops Caffe2Backend::CreateSlice(OnnxNode* onnx_node, int opset_version) {
...
 
+Caffe2Ops Caffe2Backend::CreateBatchNormalization(OnnxNode* onnx_node, int opset_version) {
+  const auto& node = onnx_node->node;
+  if (opset_version < 6) {
+    auto& attributes = onnx_node->attributes;
+    attributes.remove("consumed_inputs");
+  }
+
+  auto c2_op = CommonOnnxNodeToCaffe2Ops(onnx_node, opset_version);
+  return c2_op;
```
Comment (bddppq) [inline caffe2/onnx/backend.cc]: nit: no need to create a c2_op variable?

File: caffe2/onnx/backend.h [hunk 2]
```diff
@@ -62,6 +62,11 @@ class OnnxAttributes {
     }
   }
 
+  bool remove(const std::string& key) {
```
Comment (bddppq) [inline caffe2/onnx/backend.h]: nit: How about renaming this to `pop` and returns the AttributeProto?

**Review by bddppq — APPROVED** [review]

**Comment by houseroad** [comment]
@caffe2bot test this please

**Review by yinghai — APPROVED** [review]
Looks good!

**Pull request merged**